jekyll/design-and-roadmap.markdown

```diff
@@ -201,7 +201,7 @@ Interested in contributing? Check out the issues tagged with [help-wanted] or [g
 - [Add rename support]
 - [Add show type hierarchy support]
 - [Show index view on the VS Code extension allowing users to browse indexed gems]
-- Remove custom bundle in favor of using bundler-compose
+- Remove composed bundle in favor of using bundler-compose
```

Comment:
Let's remove this item completely. It's not really clear that we'll be able to use bundler compose as the current state won't allow us to perform auto updates.